devito/operator/operator.py

```diff
@@ -675,11 +676,23 @@ def _emit_build_profiling(self):
         perf("Operator `%s` generated in %.2f s" % (self.name, fround(tot)))
 
         max_hotspots = 3
-        for i in sorted(timings, key=timings.get, reverse=True)[:max_hotspots]:
-            v = fround(timings[i])
-            perc = fround(v/tot*100, n=10)
-            if perc > 20.:
-                perf("- [Hotspot] %s: %.2f s (%.1f %%)" % (i.lstrip('_'), v, perc))
+        threshold = 20.
+
+        def pprint(timings, indent=''):
```

Comment:
nitpicking: `pprint` is a sympy function, with the amount of sympy we use I'd prefer another name

Reply:
I'll change it to `_emit` -- no ambiguities then